packages/regl-worldview/src/Worldview.js

```diff
@@ -205,7 +205,7 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
     const { worldviewContext } = this.state;
     const initializedData = worldviewContext.initializedData;
 
-    if (process.env.NODE_ENV === "production" || !initializedData) {
+    if ((process && process.env && process.env.NODE_ENV === "production") || !initializedData) {
```

Comment:
can probably just turn accessing the `NODE_ENV` into a utility function, i.e. `getNodeEnv()`.

lodash also has a utility called `get` where you can specify a path: `_.get(process, ['env', 'NODE_ENV'])`